docs/root/configuration/other_features/io_uring.rst

```diff
@@ -0,0 +1,26 @@
+.. _config_sock_interface_io_uring:
+
+io_uring Socket Interface
+=========================
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.network.socket_interface.v3.IoUringSocketInterface>`
+
+.. attention::
+
+  The io_uring socket interface extension is experimental and is currently under active development.
+
+io_uring is an asynchronous I/O API implemented in the Linux kernel.
+This socket interface uses [liburing](https://github.com/axboe/liburing) to integrate io_uring
+with Envoy.
```

Comment:
It would be helpful to add some info about performance/latency/cpu/memory/etc tradeoffs that we expect to have when this extension is complete.  Or at least some mention of the potential which justifies the development of this extension.